pkg/workloadapi/workload.proto

```diff
@@ -215,6 +216,33 @@ enum TunnelProtocol {
   // Future options may include things like QUIC/HTTP3, etc.
 }
 
+// NativeTunnel describes how a workload (application or gateway)
+// can consume tunnel connections directly.
+message NativeTunnel {
+  // A target natively handles this type of traffic.  
+  GatewayProtocol protocol = 1;
+  // optional: if set, traffic should be sent to this port after the last zTunnel hop
+  uint32 port;
+}
+
+enum GatewayProtocol {
```

Comment:
Not a big fan of hardcoded enum for protocols. 

Any reason to not use a string ?